src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -988,9 +988,9 @@ export class ResultsViewPageStore {
 
 
     readonly isSampleAlteredMap = remoteData({
-        await: () => [this.oqlFilteredCaseAggregatedDataByUnflattenedOQLLine, this.samples],
+        await: () => [this.oqlFilteredCaseAggregatedDataByUnflattenedOQLLine, this.samples, this.coverageInformation, this.selectedMolecularProfiles],
```

Comment:
@adamabeshouse does it make sense to exclude non profile samples here?  would that be assumed by a consumer of isSampleAlteredMap?  It seems like it makes sense.     If a sample is not profiled according to currently selected molecular profile, we should not consider it not-altered, right?  Or should value type be updated to include different states, i.e. true, false, unprofiled?   Just making sure we're doing the right thing here.

Reply:
I think you have a point and it would be future-proof and most complete to do it like that, with an enum like Altered, Unaltered, Unprofiled